internal/action/main_test.go

```diff
@@ -0,0 +1,291 @@
+package action
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/hashicorp/go-tfe"
+	"github.com/stretchr/testify/assert"
+)
+
+var workspacePrefix string = "action-test"
+
+func NewTestRunConfig() *RunConfig {
```

Comment:
Make test funcs unexported. Go should prevent this from being used in another package b/c it's in a `*_test.go` file. And I think a production build too. But IIRC you can still call it from source and a test build/run will pass. Within the package making it unexported makes it more clear it is not suitable to be called outside of tests.

Reply:
thats neat!